src/identity/components/OrganizationListTab/OrganizationCard.scss

```diff
@@ -7,6 +7,16 @@
   border-radius: 2px;
 }
 
+.account--organizations-tab-suspended-orgs-card {
+  margin-top: $cf-space-3xs;
+  margin-bottom: $cf-space-3xs;
+  border-radius: 2px;
+  background-color: #0e0e0e;
+  div:not(:last-child) {
```

Comment:
The background color looks a little greyer and less navy than what's in the figma - how does this one look to you? 
bgcolor: #11121A
foreground: #7C7C81

Reply:
I see no change when setting the color to #7C7C81 😅

Reply:
Might be my monitor. Let's see if product agrees on color - if we need to make changes we can do so behind the flag.